packages/main/src/Interfaces.ts

```diff
@@ -10,11 +14,14 @@ const IAvatar = "sap.ui.webc.main.IAvatar";
 /**
  * Interface for components that may be slotted inside <code>ui5-breadcrumbs</code> as options
  *
- * @name sap.ui.webc.main.IBreadcrumbsItem
- * @interface
  * @public
  */
-const IBreadcrumbsItem = "sap.ui.webc.main.IBreadcrumbsItem";
+interface IBreadcrumbsItem {
```

Comment:
As it turned out, in a recent sync with the webcomponents team it was decided to have the interfaces of the non abstract components described with their public APIs. Abstract element/components on the other hand may not necessary implement an interface as they don't bring visual representation and extending them will not be of any value to consumers.

So this interface is not needed and its usages could be replaced with the BreadcrumbsItem type directly.